migen/build/xilinx/common.py

```diff
@@ -116,9 +116,35 @@ class XilinxDDROutputS6:
     def lower(dr):
         return XilinxDDROutputImplS6(dr.i1, dr.i2, dr.o, dr.clk)
 
+# Took this from litex/build/xilinx/common.py
+# migen doesn't include a DDR Input for S6
```

Comment:
Well it does after merging this, so this comment should be removed.

Reply:
Oh yea, removed.